src/index.js

```diff
@@ -87,17 +87,13 @@ const calculateUniqueItems = (items) => items.length;
 
 export function CartProvider({
   children,
-  id,
+  id = Math.random().toString(36).substring(7),
```

Comment:
Switching to a longer string

```suggestion
  id = [...Array(12)]
    .map((i) => (~~(Math.random() * 36)).toString(36))
    .join(""),
```